src/Components/Components/src/Sections/SectionContent.cs

```diff
@@ -37,11 +44,54 @@ void IComponent.Attach(RenderHandle renderHandle)
     }
 
     Task IComponent.SetParametersAsync(ParameterView parameters)
+    {
+        // We are not using parameters.SetParameterProperties(this)
+        // because IsDefaultContent is internal property and not a parameter
+        SetCustomParameters(parameters);
+
+        if (SectionName is not null && SectionId is not null)
+        {
+            throw new InvalidOperationException($"Both '{nameof(SectionName)}' and '{nameof(SectionId)}' cannot have values.");
```

Comment:
Would it be helpful to show _which_ values there are, so a developer can more easily find where their error is? Or does the exception somehow already indicate exactly where the error is?

Reply:
No need. It is clear where the exception is. Example:

```razor
<SectionContent SectionName="test4" SectionId="new()">
        <p>Test4</p>
    </SectionContent>
```

Reply:
No, you cannot see what file caused the error. I see now what you mean. It might be helpful to provide the values in the error message so that you can search for that values in your code. But that value might be a generic one and be in a lot of places in your code. You might as well search for "SectionContent" in your code and find the one that has both SectionId and SectionName. 
There is another error message for the case when both SectionId and SectionName are null so you would search your code for "SectionContent" in that case too.
I should probably indicate that the error is in SectionContent. Thanks!

Reply:
Oh yeah I meant something that would help the user find where in the code the error is coming from. Up to you if that is something we can reasonably do, and if it would even be helpful.

Reply:
IMO, it's fine to not include the parameter values in the exception message. The issue will almost always be that the customer's Razor code specifies both parameters on a syntactic level, as Surayya showed in her example. It would be a bit weird for a customer to specify both parameters and somehow have custom logic ensuring that exactly one of them is null at a time. So, I think it's fine to word the exception message in a manner that suggests that the two parameters shouldn't both be specified, because after that contract is broken, the values of the parameters aren't super relevant. Just my two cents, and it's probably fine either way :)